sentry/src/main/java/io/sentry/cache/CacheStrategy.java

```diff
@@ -218,7 +219,10 @@ private void moveInitFlagIfNecessary(
     return null;
   }
 
-  private @Nullable Session getFirstSession(final @NotNull SentryEnvelope envelope) {
+  private @Nullable Session getFirstSession(final @Nullable SentryEnvelope envelope) {
```

Comment:
As a private method can't we make this NotNull and check at the caller?

Reply:
getFirstSession is only called if `isValidEnvelope` validates it, so it should be not nullable already

Reply:
Fixed